src/Components/Form/index.jsx

```diff
@@ -16,14 +25,14 @@ function Form (props) {
         <form onSubmit={handleSubmit}>
           <label >
             <span>URL: </span>
-            <input name='url' type='text' />
+            <input name='url' type='text' onChange={(event) => setUrl(event.target.value)}/>
             <button type="submit">GO!</button>
           </label>
           <label className="methods">
-            <span id="get">GET</span>
-            <span id="post">POST</span>
-            <span id="put">PUT</span>
-            <span id="delete">DELETE</span>
+            <span id="get" onClick={handleClick} data-testid='test-get'>GET</span>
+            <span id="post" onClick={handleClick} data-testid='test-post'>POST</span>
+            <span id="put" onClick={handleClick} data-testid='test-put'>PUT</span>
+            <span id="delete" onClick={handleClick} data-testid='test-minus'>DELETE</span>
           </label>
```

Comment:
You're tracking which method is selected with state, but the user can't tell that. Add some styling so that you can see which method is selected.